test/quantization/core/experimental/test_float8.py

```diff
@@ -217,12 +255,69 @@ def test_cast_round_trip(self, dtype, get_input, device):
         """Numerical test of float8 conversion, by performing a round-trip cast
         to the float8 dtype and back to float32, comparing against simulated
         lower precision."""
+        if dtype is torch.float8_e8m0fnu:
+            return unittest.skip("numerics for e8m0fnu are tested elsewhere")
```

Comment:
I looked at testing e8m0 here and decided to keep it separate to keep things simple